vendor/views/vendor.py

```diff
@@ -344,6 +344,7 @@ class ProductsListView(LoginRequiredMixin, ListView):
     template_name = 'vendor/purchase_list.html'
 
     def get_queryset(self):
+        # return self.request.user.customer_profile.get(site=get_current_site(self.request)).get_completed_receipts()
         return self.request.user.customer_profile.get(site=get_current_site(self.request)).receipts.filter(status__gte=PurchaseStatus.COMPLETE)
```

Comment:
@renderbox whoa boy. okay - unrelated - but how will other open source packages that use the sites.shortcuts get_current_site(request) method reliably get the correct site based on a site alias?

Reply:
Hmm I guess we'll just have to sub-class this view from trainingcamp and change the queryset

Reply:
Got it but I forgot to remove that commented line. I was trying different queries.